flexget/plugins/api_tmdb.py

```diff
@@ -29,14 +33,20 @@ def get_tmdb_config():
     """Gets and caches tmdb config on first call."""
     global _tmdb_config
     if not _tmdb_config:
-        _tmdb_config = tmdb_request('configuration')
+        _tmdb_config = tmdb_request('configuration').json()
     return _tmdb_config
 
 
+def tmdb_post(endpoint, **params):
```

Comment:
i'd change `tmdb_request` to take `method` param and the do `return requests.request(full_url, method=method, params=params)` to avoid duplicate code paths.